crates/storage/provider/src/transaction.rs

```diff
@@ -3,47 +3,6 @@ use reth_primitives::{BlockHash, BlockNumber, H256};
 use reth_trie::StateRootError;
 use std::fmt::Debug;
 
-/// An error that can occur when using the transaction container
-#[derive(Debug, PartialEq, Eq, Clone, thiserror::Error)]
-pub enum TransactionError {
-    /// The transaction encountered a database error.
-    #[error(transparent)]
-    Database(#[from] DbError),
-    /// The transaction encountered a database integrity error.
-    #[error(transparent)]
```

Comment:
is this now fully submerged into ProviderError?

Reply:
yes